web/src/app/lists/PaginatedList.js

```diff
@@ -148,8 +156,11 @@ const mapStateToProps = state => {
 @connect(mapStateToProps)
 export class PaginatedList extends React.PureComponent {
   static propTypes = {
-    // headerNote will be displayed at the top of the list.
-    headerNote: p.oneOfType([p.string, p.element]),
+    // cardHeader will be displayed at the top of the card
+    cardHeader: p.node,
```

Comment:
Does this replace `listHeader` if used?

Reply:
You could have both if you wanted to, 1 would be rendered at the top of the card, and the other would be rendered as the first list item with text